src/Donors/ListTable/Columns/DonationCountColumn.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Give\Donors\ListTable\Columns;
+
+use Give\Donors\Models\Donor;
+use Give\Framework\ListTable\ModelColumn;
+
+/**
+ * @unreleased
+ *
+ * @extends ModelColumn<Donor>
+ */
+class DonationCountColumn extends ModelColumn
+{
+
+    protected $sortColumn = 'totalDonations';
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     */
+    public static function getId(): string
+    {
+        return 'donationCount';
+    }
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     */
+    public function getLabel(): string
+    {
+        return __('Donations', 'give');
+    }
+
+    /**
+     * @unreleased
+     *
+     * @inheritDoc
+     *
+     * @param Donor $model
+     */
+    public function getCellValue($model): string
+    {
+        $totalDonations = $model->totalDonations();
+        $label = __('No donations', 'give');
+
+        if ($totalDonations > 0) {
+            $label = sprintf(
+                _n(
+                    '%1$s donation',
+                    '%1$s donations',
+                    $totalDonations,
+                    'give'
+                ),
+                $totalDonations
+            );
+        }
```

Comment:
Let's turn this into a ternary conditional so it's more clear

Reply:
Resolved: f4f30a2e25c887daf8683c987e76db8ee8acd658